frameworks/Scala/snunit/benchmark_config.json

```
@@ -0,0 +1,26 @@
+{
+  "framework": "snunit",
+  "tests": [
+    {
+      "default": {
+        "json_url": "/json",
+        "plaintext_url": "/plaintext",
+        "port": 8080,
+        "approach": "Realistic",
+        "classification": "Micro",
+        "database": "None",
+        "framework": "SNUnit",
+        "language": "Scala",
+        "flavor": "Native",
+        "orm": "None",
+        "platform": "None",
+        "webserver": "NGINX Unit",
```

In the benchmark already exist `Nginx unit`, but like platform. 
https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/PHP/php/benchmark_config.json#L109-L110

Nginx is a web server (front-end server).
But `Nginx unit` is an application server with web server included, and for me it's better to be in platform, like `vert.x` and others. 

We need to try to have it in only one classification and with the same name `unit nginx`.

@nbrady-techempower What do you think ?

---

Didn't notice that, thank you @joanhey.
So, being consistent with the PHP example like this would work?
```suggestion
        "platform": "Unit Nginx",
        "webserver": "None",
```
Then they can be changed later in a follow up PR.